slice/Ice/Metrics.ice

```diff
@@ -101,6 +100,7 @@ module IceMX
         /// @param timestamp The local time of the process when the metrics object were retrieved.
         /// @return The metrics view data.
         /// @throws UnknownMetricsView Raised if the metrics view cannot be found.
+        ["format:sliced"]
```

Comment:
@bentoi, is this necessary?

The returned type contains classes.

Reply:
Yes, but then is it ok for the caller to slice-off the derived Metrics classes and just see the base class?